src/main/java/car_racing/domain/RacingGame.java

```diff
@@ -0,0 +1,52 @@
+package car_racing.domain;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+
+public class RacingGame {
+    private final List<RacingPlayer> racingPlayers = new ArrayList<>();
+    private int curTurn = 0;
+    private final int numberOfTurns;
+
+    public RacingGame(String stringOfNames, int numberOfTurns) {
+        String[] playerNames = StringParser.parse(stringOfNames);
+        createPlayers(playerNames);
+        InputVerifier.verifyNumberIsPos(numberOfTurns);
+        this.numberOfTurns = numberOfTurns;
+    }
+
+    private void createPlayers(String[] playerNames) {
+        for (String playerName : playerNames) {
+            racingPlayers.add(new Car(playerName));
+        }
+    }
+
+    public List<RacingPlayer> getRacingPlayers() {
+        return racingPlayers;
+    }
+
+    public void runSingleTurn() {
+        for (RacingPlayer player : racingPlayers) {
+            boolean proceed = GameRule.isAbleToProceed();
+            player.proceedNextTurn(proceed);
+        }
+        this.curTurn++;
+    }
+
+    public RacingPlayer[] getWinners() {
+        Integer maxPosition = racingPlayers.stream()
+                .max(Comparator.comparingInt(RacingPlayer::getPosition))
+                .get()
+                .getPosition();
```

Comment:
stream 사용 💯 💯  아래와 같이 mapToInt()를 사용하면 get()을 쓰지 않고도 구현 가능할 것 같아요!
get()은 NPE를 일으킬 수 있으므로 항상 조심해서 써야합니다 😄 

```suggestion
        Integer maxPosition = racingPlayers.stream()
                .mapToInt(RacingPlayer::getPosition)
                .max()
                .orElse(0);
```